dojo/tools/cred_scan/parser.py

```diff
@@ -0,0 +1,67 @@
+import csv
+import io
+from datetime import datetime
+
+
+from dojo.models import Finding
+
+
+class CredScanParser(object):
+
+    def get_scan_types(self):
+        return ["CredScan Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "CredScan Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import CSV output of CredScan scan report."
+
+    def get_findings(self, filename, test):
+        content = filename.read()
+        if type(content) is bytes:
+            content = content.decode('utf-8-sig')
```

Comment:
Any reason to use the BOM encoding? I thnik when files come from the API part, the encoding will be 'utf8'

Reply:
For some reason, the first field (TimeOfDiscovery) in the csv file came in with `u'\ufeff'` at the beginning. I was using the API to test, and was not having any issues.

Reply:
I think `utf-8-sig` will handle both cases correctly, with and without BOM.